automatix/bundlewrap.py

```diff
@@ -6,6 +6,7 @@ def _generate_python_vars(self):
         locale_vars = {}
         for key, value in self.env.systems.items():
             locale_vars[f'{key}_node'] = self.env.config["bw_repo"].get_node(value)
+            locale_vars['BW_REPO'] = self.env.config["bw_repo"]
```

Comment:
Vielleicht lieber mit einem get(), falls der Automatix ohne BW benutzt wird?

Reply:
Na dann landen wir ja gar nicht in dem Objekt. Das ist ja das BundleWrapCommand

Reply:
Stimmt natürlich :+1: